x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/utils/indices.ts

```diff
@@ -133,3 +133,7 @@ export function ingestionMethodToText(ingestionMethod: IngestionMethod) {
     defaultMessage: 'API',
   });
 }
+
+export function isHiddenIndex(index: ElasticsearchIndexWithIngestion | undefined): boolean {
+  return index !== undefined && index.name.startsWith('.');
```

Comment:
This logic is not correct for hidden indices: many hidden indices don't start with a ., and there are one or two hidden indices that do start with a '.' but aren't hidden. Really if we need to know whether an index is hidden, the server needs to add that information to the indices APIs.

Reply:
I was wondering about that, thanks.